torchprime/torch_xla_models/README.md

```diff
@@ -71,7 +72,8 @@ tp run torchprime/torch_xla_models/train.py \
     task.global_batch_size=1024 \
     dataset=wikitext \
     dataset.hf_dataset_config_name=wikitext-103-raw-v1 \
-    profile_step=6 \
+    profile_start_step=6 \
+    profile_end_step=16 \
     profile_duration=20000 \
```

Comment:
We should remove all the mentions and handling for `profile_duration` if we're migrating to explicit end steps.